homeassistant/components/verisure.py

```diff
@@ -116,15 +121,24 @@ def setup(hass, config):
 
 def reconnect():
     """ Reconnect to verisure mypages. """
+    if not LOCK.acquire(blocking=False):
+        return
+    global PASSWORD_RETRIES
     try:
-        time.sleep(1)
+        time.sleep(10)
```

Comment:
If this is called as part of a sensor updating, it will block all sensors from updating for 10 seconds. Why not just have it reconnect the next time we poll?